src/bp/ui-studio/src/web/views/FlowBuilder/index.jsx

```diff
@@ -47,16 +47,19 @@ class FlowBuilder extends Component {
 
   componentWillReceiveProps(nextProps) {
```

Comment:
Can you replace the deprecated method ? It would be nice to replace when whenever we need to edit code inside them